src/views/Orders.vue

```diff
@@ -4,18 +4,21 @@
       <ion-toolbar>
         <ion-title>{{ currentFacility.name }}</ion-title>
       </ion-toolbar>
+
       <ion-toolbar>
-        <ion-segment v-model="segmentSelected" @ionChange="segmentChanged">
-          <ion-segment-button value="open">
-            <ion-label>{{ $t("Open") }}</ion-label>
-          </ion-segment-button>
-          <ion-segment-button value="packed">
-            <ion-label>{{ $t("Packed") }}</ion-label>
-          </ion-segment-button>
-        </ion-segment>
+        <div class="header">
```

Comment:
```suggestion
        <div class="find">
```

Reply:
I have removed header class and used css selector.